File: bubbleSort/bubbleSort.js
const bubbleSort = (list) => {
  for (let i = 0; i < list.length; i++) {
    console.log(list);
    for (let j = 0; j < list.length - i - 1; j++) {
      if (list[j] > list[j + 1]) {
        let help = list[j];
        list[j] = list[j + 1];
        list[j + 1] = help;
      }
    }
  }
  return list;
};

const main = () => {
  const list = [55, 33, 23, 64, 13, 2, 6, 43];
  console.log(bubbleSort(list));
};

main();

File: bubbleSort/bubbleSort.py
import random

def bubbleSort(list):
    n = len(list)
    for i in range(n):
        print(list)
        for j in range(0,n-i-1):
            if list[j] > list[j+1]:
                list[j], list[j+1] = list[j+1], list[j]
    return list

if __name__ == '__main__':
    tamano_de_lista = int(input('De que tamano sera la lista?'))

    list = [random.randint(0,100) for i in range(tamano_de_lista)]
    # print(list)

    lista_ordenada = bubbleSort(list)
    # print(lista_ordenada)
